documentation/modules/installing-mtv-operator.adoc

```diff
@@ -34,13 +34,15 @@ The {operator-name} is a Community Operator. Red Hat does not support Community
 ====
 endif::[]
 . Click *{operator-name-ui}* and then click *Install*.
-. On the *Install Operator* page, click *Install*.
-. Click *Operators* -> *Installed Operators* to verify that *{operator-name-ui}* appears in the *{namespace}* project with the status *Succeeded*.
-. Click *{operator-name-ui}*.
-. Under *Provided APIs*, locate the *ForkliftController*, and click *Create Instance*.
+. Click *Create ForkLift Controller* when the button becomes active.
 . Click *Create*.
++
+Your ForkLift Controller appears in the list that is displayed.
 . Click *Workloads* -> *Pods* to verify that the {project-short} pods are running.
-include::snippet_getting_web_console_url_web.adoc[]
+. Click *Operators* -> *Installed Operators* to verify that *{operator-name-ui}* appears in the *{namespace}* project with the status *Succeeded*.
++
+The (product-short) web console is automatically installed in the {ocp} web console. You are ready to defining and running migrations.
```

Comment:
```suggestion
The "Migration" menu item is automatically add in the {ocp} web console left side navigation bar. You are ready to defining and running migrations.
```

Reply:
Just to tidy up the above suggestion: 
The "Migration" menu item is automatically added to the navigation bar, displayed on the left of the {ocp} web console. You are ready to define and run migrations.